account_fiscal_position_autodetect_optional_vies/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+Create a new partner and define some country.
```

Comment:
The first two steps should go in CONFIGURE.rst, and please put them with the bullet list